core/src/main/java/org/wildfly/extras/sunstone/api/impl/ObjectType.java

```diff
@@ -6,7 +6,8 @@
  */
 public enum ObjectType implements ObjectPropertiesType {
     CLOUD_PROVIDER("cloud.provider", "cloud provider"),
-    NODE("node", "node");
+    NODE("node", "node"),
+    JUNIT5("junit5", "JUnit5");
```

Comment:
I think we talked about it before. But I still fail to understand how conceptually is `junit` in same category as eg `node`

Reply:
I mean junit5 is implementation detail. What will se do once  junit6 will be out? :)